src/transformers/models/esm/tokenization_esm.py

```diff
@@ -92,11 +90,10 @@ def _convert_token_to_id(self, token: str) -> int:
     def _tokenize(self, text, **kwargs):
         return text.split()
 
-    def get_vocab_size(self, with_added_tokens=False):
-        return len(self._id_to_token)
-
     def get_vocab(self):
-        return {token: i for i, token in enumerate(self.all_tokens)}
+        base_vocab = {token: i for i, token in enumerate(self.all_tokens)}
```

Comment:
```suggestion
        base_vocab = self._token_to_id.copy()
```

Reply:
you don't necessarily have to recompute it

Reply:
Good improvement - added!